.github/workflows/release.yml

```diff
@@ -54,5 +55,5 @@ jobs:
           version: pnpm run version
           publish: pnpm run publish
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.VERSION_PACKAGES_TOKEN }}
```

Comment:
Can you just add a YAML comment with links to what you posted in this PR description saying why we need this?